requirements.txt

```diff
@@ -118,12 +117,12 @@ pytz==2021.3; python_version >= "3.6" and python_full_version < "3.0.0" or pytho
 pyxb==1.2.5
 pyyaml==6.0; python_version >= "3.6"
 razorpay==1.2.0
-redis==4.0.2; python_version >= "3.6"
```

Comment:
What happened here (downgrade)?

Reply:
Seems there was a breaking change and django-redis downgraded it for now: https://github.com/jazzband/django-redis/pull/570

Reply:
Ah, makes sense. Thanks for elaborating.